orcestra_book/data_policy.md

```diff
@@ -0,0 +1,11 @@
+# Data Policy
+
+ORCESTRA encourages free use and access to data without strict control. We trust in the integrity of the scientific community to honor the contributions of those who collected these data.
+
+The ORCESTRA data are a product of the collective efforts and funding of various independent research groups. It often represents years of work in developing, certifying, and operating instruments, often tied to specific research or Ph.D. projects. We ask users to respect the priority of the data collectors by coordinating use with the instrument Principal Investigators (PIs) and by clearly acknowledging the source of the data in any publications or presentations.
+
```

Comment:
```suggestion
Specific data may be bound by agreements associated with the platforms from which they were collected, for instance the data policies of the German Research Vessels, or HALO.

```

Maybe we should add this one line too, although it is a bit contradictory. Anyhow, we probably need to include the data policies of the platforms somehow...